docs/README.md

```diff
@@ -0,0 +1,946 @@
+                        Folder containing the special model-metadata.json file (https://github.com/Kaggle/kaggle-api/wiki/Model-Metadata). Defaults to current working directory
+```
+
+Example:
+
+`kaggle models create -p /path/to/model`
+
+##### Delete model
+
+```
+usage: kaggle models delete [-h] [model]
+
+required arguments:
+  model       Model URL suffix in format <owner>/<model-name>
+
+optional arguments:
+  -h, --help  show this help message and exit
+```
+
+Example:
+
+`kaggle models delete tensorflow/toxicity`
+
+##### Update a model
+
+If you want to update a model, you need a metadata file at first. You can fetch the data by running `kaggle models get owner/slug -p folder`.
+
+```
+usage: kaggle models update [-h] [-p FOLDER]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -p FOLDER, --path FOLDER
+                        Folder containing the special model-metadata.json file (https://github.com/Kaggle/kaggle-api/wiki/Model-Metadata). Defaults to current working directory
+```
+
+Example:
+
+`kaggle models update -p /path/to/model`
+
+#### Model Instances
+
+The API supports the following commands for Kaggle Model Instances.
+
+```
+usage: kaggle models instances [-h]
+                             {init, create, delete, update} ...
+
+optional arguments:
+  -h, --help            show this help message and exit
+
+commands:
+  {get, init, create, delete}
+    get                 Get a model instance
+    init                Initialize metadata file for model instance creation
+    create              Create a new model instance
+    delete              Delete a model instance
+    update              Update a model instance
+```
+
+##### Get model instance
+
+```
+usage: kaggle models instances get [-h] [-p FOLDER] [modelInstance]
+
+required arguments:
+  modelInstance         Model Instance URL suffix in format <owner>/<model-name>/<framework>/<instance-slug>
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -p PATH, --path PATH  Folder where the special model-instance-metadata.json file (https://github.com/Kaggle/kaggle-api/wiki/Model-Metadata) will be downloaded (if specified).
+```
+
+Example:
+
+`kaggle models instances get tensorflow/toxicity/tfjs/default`
+
+##### Initialize metadata file for a model instance
+
+```
+usage: kaggle models instances init [-h] [-p FOLDER]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -p FOLDER, --path FOLDER
+                        Folder to create the model-instance-metadata.json file (https://github.com/Kaggle/kaggle-api/wiki/Model-Metadata). Defaults to current working directory
+```
+
+Example:
+
+`kaggle models instances init -p /path/to/modelinstance`
+
+##### Create a new model instance
+
+If you want to create a new model instance, you need to initiate metadata file at first. You could fulfill this by running `kaggle models instances init` as describe above.

```

Comment:
Similar to before, I think "initiate" should be "initialize the" and "fulfill" should be "do". Also "describeD".